tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -90,6 +97,7 @@ class PipelineParallelForwardBackwardTestBase:
     GLOBAL_BATCH_SIZE = 16
     MICRO_BATCH_SIZE = 2
     HIDDEN_SIZE = 32
+    NUM_EPOCHS = 1
```

Comment:
Nit: Is this an Epoch or a fwd-bwd iteration?

Reply:
I believe it is both. We can name it `NUM_FWD_BWD_ITERATIONS`. What do you think?